addon/services/keyboard.js

```diff
@@ -68,6 +68,11 @@ export default Service.extend({
   },
 
   _respond(event) {
+    // if (!this.isPropagationEnabled) {
+    //   event.preventDefault();
+    //   event.stopPropagation();
+    // }
+
```

Comment:
A little dead code.

Reply:
Don't worry about this; you've done enough. I'll fix it during the release.

Reply:
thanks a ton!!!